fastpass/src/main/scala/scala/meta/internal/fastpass/bazelbuild/Bazel.scala

```diff
@@ -181,10 +186,25 @@ class Bazel(bazelPath: Path, cwd: Path) {
         // Use textproto instead of proto, see https://github.com/bazelbuild/bazel/issues/12984
         "--output=textproto"
       )
-      val code = run(cmd, out, err)
+
+      val code = {
+        val errByteArrayOutputStream = new ByteArrayOutputStream
+        val code = run(cmd, out, errByteArrayOutputStream)
+        if (
+          retries > 0 && errByteArrayOutputStream
+            .toString()
+            .contains(bazelInternalError)
```

Comment:
I think we don't need to look at the error stream if the exit code was 0?
```suggestion
          code != 0 && retries > 0 && errByteArrayOutputStream
            .toString()
            .contains(bazelInternalError)
```